railties/test/commands/credentials_test.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+require "isolation/abstract_unit"
+require "env_helpers"
+require "rails/command"
+require "rails/commands/credentials/credentials_command"
+
+class Rails::Command::CredentialsCommandTest < ActiveSupport::TestCase
+  include ActiveSupport::Testing::Isolation, EnvHelpers
+
+  def setup
+    build_app
+  end
+
+  def teardown
+    teardown_app
```

Comment:
Missing `super` in both setup and teardown. Let's go with either `setup { build_app }` or `setup :build_app`